### DISCUSSION TITLE
landmark and camera poses didn't have same coordinate?

### DISCUSSION THREAD

**Discussion by fangxu622** [stella-cv/stella_vslam#172] [Q&A] [answered]
when I visualize the landmark( extract from msg file) and camera pose, they didn't in the same coordinate.  how to made them in one coordinate ?

 aist_living_lab1 example data 

![image](https://user-images.githubusercontent.com/16681534/135756024-ea000788-b762-471f-8410-af86b1b6289a.png)

**Comment by ymd-stella** [comment]
See https://github.com/OpenVSLAM-Community/openvslam/discussions/105

**Comment by fangxu622** [comment, marked as answer]
I see. But now I want to visualize them in one coordinate. how to modify it in source code?

**Reply by ymd-stella** [reply]
See https://gist.github.com/ymd-stella/a60edd4242dd99ee4bcbd0747031d524#file-read_msg_open3d_plot-py-L20-L27.
